### DISCUSSION TITLE
How to provide flags to chrome?

### DISCUSSION THREAD

**Discussion by klmntv** [wxt-dev/wxt#331] [General]
I want to run chrome with specified flags, such as  --remote-debugging-port=9222. How can i do that?

**Comment by aklinker1** [comment]
See https://wxt.dev/guide/development.html#configure-browser-startup

Create a file called `web-ext.config.ts`, and make the contents look like this:

```ts
import { defineRunnerConfig } from 'wxt';

export default defineRunnerConfig({
  chromiumArgs: ["--remote-debugging-port=9222"]
});
```

When running `wxt` to start the browser in dev mode, it will use this file to configure the browser.

**Reply by klmntv** [reply]
Thank you! It works, but I get this message in console after browser start 

`✖ Command failed after 26.4 s                                                                                                                                       10:18:28 AM

 ERROR  connect ECONNREFUSED 127.0.0.1:64609                                                                                                                         10:18:28 AM

  at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1247:16)
`

It looks like HMR is working, but connection is lost after a certain period of time.

**Reply by aklinker1** [reply]
Unfortunately, I don't know much about debugging, there's some info on https://github.com/wxt-dev/wxt/issues/52 and it's related issues.

**Comment by ZinkNotTheMetal** [comment]
Any update on this? I'm trying to run a few sessions with playwright and need to expose a debugging port
